zone/bot.cpp

```diff
@@ -2293,21 +2292,21 @@ void Bot::AI_Process()
 		const EQ::ItemInstance* p_item = GetBotItem(EQ::invslot::slotPrimary);
 		const EQ::ItemInstance* s_item = GetBotItem(EQ::invslot::slotSecondary);
 
-		CombatRangeInput input = {
-			.target = tar,
-			.target_distance = tar_distance,
-			.stop_melee_level = stop_melee_level,
-			.p_item = p_item,
-			.s_item = s_item
+		CombatRangeInput combat_range_input = {
+			.target                 = tar,
+			.target_distance        = tar_distance,
+			.stop_melee_level       = stop_melee_level,
+			.p_item                 = p_item,
+			.s_item                 = s_item
 		};
 
-		CombatRangeOutput o = EvaluateCombatRange(input);
+		CombatRangeOutput combat_range_output = EvaluateCombatRange(combat_range_input);
```

Comment:
Generally when we declare variables that are a receiver of a complex type (class, struct etc.), adding verbosity doesn't do much to help readability because its members or fields describe it already. That's why I had just `o` before. 

Simple types it makes sense to describe more of what it does.